app/controllers/users/sessions_controller.rb

```diff
@@ -225,6 +225,8 @@ def auth_callback
 
           if onboard_user
```

Comment:
@yash-learner When `onboard_user` is `:server_limit_reached`, it counts as a _truthy_ value, so this condition will pass, and the success message will be flashed.

Don't mix types. If you're going to use symbols to define different return states, return only symbols from service.